### ISSUE TITLE
Story from our student

### ISSUE THREAD

**Issue by ebunayo** [Bloceducare/web3bridgewebsite#106]
Since all our students are developers, I want to suggest we have each one submit a PR adding their own testimonial and we can merge to keep the data increasing. @Goodness5

**Comment by psychemist** [comment]
Do we use an in-house form where all cohort graduates can fill in their details and submit to be added in the testimonials page instead of processing hundreds of PRs per cohort? @ebunayo

**Comment by ebunayo** [comment]
> Do we use an in-house form where all cohort graduates can fill in their details and submit to be added in the testimonials page instead of processing hundreds of PRs per cohort? @ebunayo

Yes this makes sense, we can create a form and possibly add the response link to repo @Goodness5

**Comment by Goodness5** [comment]
Yes @ebunayo a form that saves the response data so it'll be available for rendering on the frontend testimonial section
